doc/src/devdocs/types.md

```diff
@@ -100,6 +100,7 @@ julia> dump(Array)
          lb: Core.BottomType Union{}
          ub: Any
        body: Array{T,N} <: DenseArray{T,N}
+```
```

Comment:
Dunno, none of the blocks in here are doctests. Converting some of them likely wouldn't hurt anything I would think. @JeffBezanson?